app/Http/Controllers/PressReleasesController.php

```diff
@@ -149,7 +149,7 @@ protected function getNavElements($title)
 
     public function show($id)
     {
-        $item = $this->repository->getById((int) $id);
+        $item = $this->repository->published()->findOrFail((int) $id);
```

Comment:
@nikhiltri the `getById` method is a twill model method and not an eloquent one so I replaced it with the findOrFail which will 404 👍

Reply:
Nice! Much better.